studioShared/deskStructure.ts

```diff
@@ -0,0 +1,17 @@
+import { i18n } from "languages";
+import { StructureResolver } from "sanity/structure";
+import { customerCaseID } from "./schemas/documents/customerCase";
+
+export const deskStructure: StructureResolver = (S) =>
+  S.list()
+    .title("Content")
+    .items([
+      S.listItem()
+        .title("Customer cases")
+        .child(
+          S.documentTypeList(customerCaseID)
```

Comment:
I think the type in `documentTypeList(customerCaseID)` is ignored when using `.filter(...)`, which is why we have to check type in filter as well. So we might want to use a plain `documentList()` to avoid confusion (I at least wondered why you needed the extra `_type == $type`).

Reply:
Yes! Good catch